### ISSUE TITLE
Improve secrets with attribute randrange(n, m). Part two

### ISSUE THREAD

**Issue by NotwhatIwanttotell** [python/cpython#102689]
[Improve secrets with attribute randrange(n, m) #101064](https://github.com/python/cpython/issues/101064) got closed. I am opening up the subject again with a new angle:

The scope of the secrets module is to get the highest standard of randomness. It is supposed to have a "better randomness" than the standard random module.

So lets give you an example where the highest standard of randomness within an interval is necessary:
BIP 39 - Wordlist!

Its 2048 words used to create wallets. It starts with one (1) and not zero (0) so secrets.randbelow(2048) will not work as zero is not an option. The only way to make it work is secrets.randrange(1, 2048).

So in short; secrets.randrange(n, m) is a logical solution to BIP39 as it has a higher randomness than random.randrange(n,m).

https://github.com/bitcoin/bips/blob/master/bip-0039/english.txt

**Comment by stevendaprano** [comment]
Please don't open a new issue when the previous issue has been closed. There is no reason to split the discussion. Please make any replies on the previous issue, and respect the fact that a core developer has closed it. Trying to persuade us to reopen the previous issue is fine. Spamming the bug tracker with new issues for the same feature is not.
